pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/BundleData.java

```diff
@@ -103,4 +97,13 @@ public int getTopics() {
     public void setTopics(int topics) {
         this.topics = topics;
     }
+
+    @Override
+    public int compareTo(BundleData o) {
+        int result = this.longTermData.compareTo(o.longTermData);
+        if (result == 0) {
+            result = this.shortTermData.compareTo(o.shortTermData);
+        }
```

Comment:
What if short-term data is significantly higher than long-term's?

Reply:
May be we should prefer to update short-term data than long-term data?
WDYT, @hangc0276 @codelipenghui

Reply:
I prefer to compare the `shortTermData` first.

Reply:
May be we can compare the `shortTermData` first?
WDYT.  @heesung-sn

Reply:
I think this may require some community agreement, but +1 for short-term IMO.

Reply:
+1 for short-term data